pkgs/tools/networking/castnow/default.nix

```diff
@@ -20,7 +20,7 @@ buildNpmPackage rec {
 
   meta = {
     description = "Command-line Chromecast player";
-    homepage = "commandline chromecast player";
+    homepage = "https://github.com/${src.owner}/${src.repo}";
```

Comment:
I would use a string literal here, just in case in `src` updates.

Reply:
Hi! Sorry for answering so late, it seems I've missed the notification.

What do you mean by strings literal in that context ? I'm not sure I see how `src` update could cause issues, if you could explain :).

Reply:
Writing a link directly to the homepage makes it easier to click and copy. (Anyway the link is already in `src.meta.homepage`. However, we do not recommend its use yet. We may automatically add this entry in `meta.repository` later.)